src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderLabel.cs

```diff
@@ -10,17 +10,14 @@ namespace System.Security.Cryptography.Cose
     [DebuggerDisplay("{DebuggerDisplay,nq}")]
     public readonly struct CoseHeaderLabel : IEquatable<CoseHeaderLabel>
     {
-        internal string LabelName => LabelAsString ?? LabelAsInt32.ToString();
+        internal string LabelName => $"\"{LabelAsString}\"" ?? LabelAsInt32.ToString();
```

Comment:
`$"\"{LabelAsString}\""`

This expression can never be null, so the `?? LabelAsInt32.ToString();` will never happen.

```suggestion
        internal string LabelName => LabelAsString is null ? LabelAsInt32.ToString() ? $"\"{LabelAsString}\"";
```